extensions/amp-consent/0.1/test/test-consent-ui.js

```diff
@@ -132,14 +146,17 @@ describes.realWin('consent-ui', {
       expect(parent.classList.contains('amp-hidden')).to.be.true;
     });
 
-    it('append/remove iframe', () => {
+    it('append/remove iframe', function* () {
       const config = dict({
         'promptUISrc': 'https//promptUISrc',
       });
       consentUI =
           new ConsentUI(mockInstance, config);
       expect(elementByTag(parent, 'iframe')).to.be.null;
       consentUI.show();
+      yield macroTask();
+      yield macroTask();
```

Comment:
nit: can you try this with one `yield macroTask()` ? I've found this works with just one and doesn't need 3? 🤔

Reply:
Ahhh! Thank you for catching this. This was my attempt to fix the test, but it didn't work out. I ended up mocking the getConsentInstanceInfo from consent state manager.